src/main/java/Duke.java

```diff
@@ -1,10 +1,183 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    // types of command
+    enum COMMANDS {
+        COMMAND_INIT,
+        COMMAND_ADD,
+        COMMAND_LIST,
+        COMMAND_MARKDONE,
+        COMMAND_EXIT,
+        COMMAND_UNRECOGNIZE,
+    }
+
+    // Formatting
+    private final static String formatDashes = "----------------------------------------------------------------------";
+    private final static String formatTwoTabs = "    ";
+    private final static String formatFourTabs = "        ";
+
+    // String object to hold any user input or output message
+    private static String message = "";
+    private static String[] splitMessage;
+    private static String command = "";
+
+    private static Scanner in = new Scanner(System.in);
+
+    // Characters for tick and cross
+    public static char tick = '\u2713';
+    public static char cross = '\u2718';
+
+    // Boolean for exit app
+    private static boolean isExit = false;
+
+    // User task list
+    private static List<Task> taskList = new ArrayList<>();
+
+    // Print message with formatting
+    private static void reply(String msg) {
```

Comment:
It might be better to avoid abbreviation?

Reply:
Noted, thanks!